src/Heist/Splices/Json.hs

```diff
@@ -89,7 +96,20 @@ boolToText b = if b then "true" else "false"
 numToText :: ToJSON a => a -> Text
 numToText = T.decodeUtf8 . S.concat . L.toChunks . encode
 
+#if MIN_VERSION_aeson(2,0,0)
+------------------------------------------------------------------------------
+findExpr :: Text -> Value -> Maybe Value
+findExpr t = go (T.split (=='.') t)
+  where
+    go [] !value     = Just value
+    go (x:xs) !value = findIn value >>= go xs
+      where
+        findIn (Object obj) = KM.lookup (K.fromText x) obj
```

Comment:
This is the only line that is different and should be in the `#if`.  It seems superfluous to cut&paste the whole function.